os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -110,6 +110,18 @@ extern uint32_t g_assertpc;
 extern struct tcb_s *g_faultmsg_sender;
 extern sq_queue_t g_faultmsg_list;
 extern sq_queue_t g_freemsg_list;
+
+#define ADD_READYTORUN_LIST(tcb)    do { \
```

Comment:
`BM_EXCLUDE_SCHEDULING` and `ADD_READYTORUN_LIST` look pair. If it is, how about using the same naming policy ?
1. BM_EXCLUDE_SCHEDULING and BM_INCLUDE_SCHEDULING
2. REMOVE_READYTORUN_LIST and ADD_READYTORUN_LIST

Reply:
This macro includes several operations including adding it in readytorun list. So, in my opinion, 1st option is better.